src/test/java/hudson/plugins/git/GitSCMSlowTest.java

```diff
@@ -340,6 +360,18 @@ public GitClient decorate(GitSCM scm, GitClient git) throws IOException, Interru
     @Test
     public void testMergeWithMatrixBuild() throws Exception {
         assumeTrue("Test class max time " + MAX_SECONDS_FOR_THESE_TESTS + " exceeded", isTimeAvailable());
+        /* The testMergeWithMatrixBuild test fails randomly on several
+         * machines when commit.gpgsign and tag.gpgsign are not
+         * enabled if the TestPreBuildMerge implementation is used. It
+         * passes consistently when PreBuildMerge is used. Rather than
```

Comment:
interesting. what sort of error is it? did it happen on https://ci.jenkins.io/?

Reply:
I ran more tests to check failure rates on various machines in my home lab.  Failure rate seemed to decrease as machine performance decreased.

* RHEL 8 with Ryzen 5 - 16/16 failed
* Ubuntu 22.04 with Xeon(R) X5660 - 14/16 failed
* Ubuntu 22.04 with Xeon(R) X3450 - 13/16 failed
* Ubuntu 22.04 with i5-6400 - 12/16 failed
* Debian testing with i5-2400 - 7/16 failed